amazon-kinesis-client/src/main/java/software/amazon/kinesis/retrieval/fanout/FanOutRecordsPublisher.java

```diff
@@ -184,29 +200,55 @@ private void handleFlowError(Throwable t) {
         }
     }
 
-    private String throwableCategory(Throwable t) {
+    private enum ThrowableType {
+        ACUIRE_TIMEOUT("AcquireTimeout"), READ_TIMEOUT("ReadTimeout"), OTHER("Other");
```

Comment:
Not that it matters too much, as it's private, but there's a typo here: `ACUIRE_TIMEOUT` -> `ACQUIRE_TIMEOUT` @sahilpalvia @pfifer

Reply:
Thanks for reporting this, we'll fix it in a later release.